include/aikido/planner/ompl/Planner.hpp

```diff
@@ -209,6 +210,51 @@ trajectory::InterpolatedPtr planCRRTConnect(
     double _minStepsize,
     double _minTreeConnectionDistance);
 
+/// Use LRA* planner to plan a path that moves from the
+/// start to a goal region given a graph to search on
+/// \param _start The start state
+/// \param _goal The goal state
+/// \param _statespace The StateSpace that the planner must plan within
+/// \param _interpolator An Interpolator defined on the StateSpace. This is used
+/// to interpolate between two points within the space.
+/// \param _dmetric A valid distance metric defined on the StateSpace
+/// \param _sampler A Sampleable that can sample states from the
+/// StateSpace. Warning: Many OMPL planners internally assume this sampler
+/// samples uniformly. Care should be taken when using a non-uniform sampler.
+/// \param _validityConstraint A constraint used to test validity during
+/// planning. This should include collision checking and any other constraints
+/// that must be satisfied for a state to be considered valid.
+/// \param _boundsConstraint A constraint used to determine whether states
+/// encountered during planning fall within any bounds specified on the
+/// StateSpace. In addition to the _validityConstraint, this must also be
+/// satsified for a state to be considered valid.
+/// \param _boundsProjector A Projectable that projects a state back within
+/// valid bounds defined on the StateSpace
+/// \param _maxDistanceBtwProjections The maximum distance (under dmetric)
+/// between projecting and validity checking two successive points on a tree
+/// extension
+/// \param _roadmapPath The path to the graph to be used
+/// \param _lookahead The number of edges over which lazy search is performed.
+/// The lookahead helps interpolate between LWA* and LazySP
+/// \param _greediness The number of edges along the lazy path to evaluate
+/// \param _maxPlanTime The maximum time to allow the planner to search for a
+/// solution
+trajectory::InterpolatedPtr planLRAstar(
```

Comment:
This function should probably be moved into a `planning::ompl::pr` namespace, so we can still build the `planning::ompl` module without requiring the user to install `pr_ompl`. This also provides a nice migration path as planners get merged into the core of OMPL: they can be promoted up a namespace.